src/main/java/com/powsybl/openloadflow/sensi/DcSensitivityAnalysis.java

```diff
@@ -605,6 +601,14 @@ private void applyInjectionContingencies(Network network, LfNetwork lfNetwork, P
         lccs.stream().map(Pair::getKey).forEach(busesToSave::add);
         vscs.stream().map(LfGenerator::getBus).forEach(busesToSave::add);
         generators.stream().map(LfGenerator::getBus).forEach(busesToSave::add);
+        for (String loadId : contingency.getLoadIdsToLose()) {
```

Comment:
For consistency reason, it might be better to have a separate method to process a single load to get a LfBus, process all the loads in contingency.getLoadIdsToLose(), and then add them to busesToSave (like what is done for generators and lccs/vscs)

Reply:
We don't have the detail of a load from a LfBus, we just keep some information on a list of loads for performance issues. So We don't store the id of a load in the LfBus.